sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -221,6 +221,15 @@ private[spark] object HiveUtils extends Logging {
       .booleanConf
       .createWithDefault(true)
 
+  val QUOTE_HIVE_STRUCT_FIELD_NAME =
```

Comment:
Do we really need this config? After this PR, I think there is no risk at all, and config protection seems not needed.

Reply:
I second this and will send a follow-up to remove it.